lib/ex_huobi/margin/rest/account.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ExHuobi.Margin.Rest.Account do
```

Comment:
Can you add unit tests for this module also ?

Reply:
Can you add the typespec for all of the public functions https://hexdocs.pm/elixir/typespecs.html